etc/fields.yml

```diff
@@ -13,10 +13,18 @@ env:
       These fields contain data about the environment in which the
       transaction was captured.
   fields:
-    - name: shipper
+    - name: beat.hostname
```

Comment:
@tsg Do you know why the packetbeat system tests running before without these changes?